web-console/src/lib/components/layout/pipelines/PipelineEditLayout.svelte

```diff
@@ -229,7 +230,7 @@ example = "1.0"`
     {#snippet afterStart()}
       <div class="flex min-w-0 flex-1 flex-col gap-x-4 gap-y-1 sm:flex-row sm:items-center">
         <PipelineBreadcrumbs
-          class="-ml-3 py-1 pl-3"
+          class="-ml-3 pb-1 pl-3"
```

Comment:
what is this?

Reply:
I thought languages are supposed to become more user-friendly, not more cryptic.
"modern"? This looks like assembly from the 1960s